artemis/modules/data/api_scanner_data.py

```diff
@@ -0,0 +1,18 @@
+from os.path import dirname
+
+# Common LFI payloads
+with open(f"{dirname(__file__)}/swagger.txt", "r") as f:
+    COMMON_SPEC_PATHS = f.read().splitlines()
+    COMMON_SPEC_PATHS = [payload.strip() for payload in COMMON_SPEC_PATHS if not payload.startswith("#")]
+
+VULN_DETAILS_MAP = {
+    "Endpoint performs HTTP verb which is not documented": "Unsupported HTTP Method",
+    "One or more parameter is vulnerable to SQL Injection Attack": "SQL Injection",
```

Comment:
From my experience, just checking for 500 status code is too prone to false positives (e.g. even if there's no SQLi, the app might crash because there's no user with username `' or 1=1`. What do you think about slightly patching offat to check also for the response time for time-based SQLi? Or other way of filtering?